plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -58,24 +58,12 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 		await page.locator( '#title' ).fill( virtualProductName );
 		await page.locator( '#_regular_price' ).fill( productPrice );
 		await page.locator( '#_virtual' ).click();
+		await page.getByRole( 'button', { name: 'Save Draft' } ).click();
+		await expect( page.locator( '#sample-permalink' ) ).toBeVisible();
 		await page.locator( '#publish' ).click();
 		await page.waitForLoadState( 'networkidle' );
 
-		// When running in parallel, clicking the publish button sometimes saves products as a draft
-		if (
-			(
-				await page.locator( '#post-status-display' ).innerText()
-			 ).includes( 'Draft' )
-		) {
-			await page.locator( '#publish' ).click();
-			await page.waitForLoadState( 'networkidle' );
-		}
-
-		await expect(
-			page
-				.locator( 'div.notice-success > p' )
-				.filter( { hasText: 'Product published.' } )
-		).toBeVisible();
```

Comment:
Is it safe to remove this? It seems this code could reduce some potential flakiness.

Reply:
> It seems this code could reduce some potential flakiness

The original intention indeed is to reduce flakiness. But when I was running the e2e tests locally, this test kept failing. I do think there could be a more conservative way of fixing the if block. I'll revert these lines and come up with a more conservative way to fix the flakiness.